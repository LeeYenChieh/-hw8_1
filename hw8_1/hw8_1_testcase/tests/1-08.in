zE988NykEpFimL4hAtFA V
fxFnyHfHo0JHT2nT3mufFiFjroIfq8Xy9 6Yohg98vXYGx5ymPZtB9
N6Pg8SK4VgPsg10P5HUYy1L CL72QEVyNU2faMiy503mmfyQYcvZ7Ri
rTijLhQVif nmt1LyEhsd6ROBX5wyRHijv5A6b3WHDduxw8ViuE
kJTY9wl5Tm0r6l0TXGypRnWsbSyHZVvUss6BlLinXRwnf0w6GApnYQ X088wfIW1XGF5jlLYoSnd6Mo9x19EORE9FY3emHB3SHyK4XPd5JQKjbHI2EMA
gqkeMP98Yy8Hs3ZQdlpEyouNIoNAm8jyMwp Nv1iSnfNf1zL4r2HBk6
opYRVEjjG EDzNFxRs8K1P1m
WD2RGU9igs7kEXy6PN5obtR7oLWXA2QrSWj9FFWCTuQjA0CDqQcAjezcZ LJXfJUDQ7m5X0lPSk6615mpVI7aRiF0Pzv77
N6Pg8SK4VgPsg10P5HUYy1L yeyKGMRPz8s7j1lKN
zE988NykEpFimL4hAtFA display
