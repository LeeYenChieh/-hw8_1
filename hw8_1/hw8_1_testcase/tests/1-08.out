2mgRh VBVmKqw6cUaWQsldfGmB6IuFGWhzENAqUnr2cCZvTuERlLvQ5CUvO8C526 1DJzUPNDjXzGkhy7P4P7qwe5uJt0jrU2S3qEu 
